byway/src/main/java/com/google/sps/servlets/StopsServlet.java

```diff
@@ -14,19 +14,34 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+/** Servlet that handles data for stops */
+@WebServlet("/api/stop")
+public final class StopsServlet extends HttpServlet {
+  ArrayList<String> stops= new ArrayList<String>();
+  private final Gson gson = new Gson(); 
+  // Implement datastore in routepage branch
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    Gson gson = new Gson();
+    String json=gson.toJson(stops);
```

Comment:
small nit: add space before and after equals sign

Reply:
Oh I did not get a notification for this nit, fixed!